packages/contracts/test/helpers/test-runner/test-runner.ts

```diff
@@ -64,24 +64,25 @@ export class ExecutionManagerTestRunner {
     StateManager: {
       owner: '$OVM_EXECUTION_MANAGER',
       accounts: {
-        ['0x4200000000000000000000000000000000000002']: {
+        [predeploys.OVM_DeployerWhitelist]: {
```

Comment:
Replacing the addresses with members of the `predeploys` object is somewhat orthogonal to the theme of this PR, but seemed like a pretty safe change to include.